couchdb_custom/couchdb_custom.py

```diff
@@ -38,6 +38,13 @@ def check(self, instance):
             session.auth = (user, password)
             context = SessionContext(host, port, local_port, session)
             node_hosts = _get_couch_nodes(context)
+            for node_host in node_hosts:
+                self.gauge(
+                    'couchdb.in_maintenance_mode',
+                    int(_in_maintenance_mode(context, node_host)),
+                    tags=instance_tags + ["node:{}".format(node_host)]
```

Comment:
Is host not automatically included with this metric and therefore a `node` tag redundant?